.gitattributes

```diff
@@ -8,3 +8,4 @@ ofrak_core/test_ofrak/components/assets/* filter=lfs diff=lfs merge=lfs -text
 ofrak_core/test_ofrak/components/assets/README.md !filter !diff !merge text
 ofrak_tutorial/assets/* filter=lfs diff=lfs merge=lfs -text
 docs/user-guide/gui/assets/* filter=lfs diff=lfs merge=lfs -text
+ofrak_core/test_ofrak/components/assets/kernel32.dll filter=lfs diff=lfs merge=lfs -text
```

Comment:
```suggestion
```

This line should not be required, since line 7 of this file should already track everything in that folder